### PR TITLE
Jellyfish

### PR DESCRIPTION
**Pull request by yaraaguilar**

### CODE REVIEW

File: marine.javascript [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (carlosrymer) [inline marine.javascript:1]: File extension should be `.js`. Do you want to add an example of an object?

**Comment by yaraaguilar** [comment]
I have added a new file called **kelp.js**.

File: kelp.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+
+// Program 1: If //
+
+let toyBabiesOfSis = 12;
+
+if (! toyBabiesOfSis >= 20)
```
Comment (carlosrymer) [inline kelp.js:6]: You may not have wanted to include the `!` here.

File: kelp.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    alert("Hmm, you haven't told me how many apps you have or what your storage space is.")
+}
+
+// Program 3: Switch //
+
+let academiaAesthetics = ["Dark academia", "Light academia", "Classical academia", "Romantic academia", "Royalty core academia", "Pastel academia"];
+
+let yourAesthetic = academiaAesthetics[5];
+
+switch(yourAesthetic)
```
Comment (carlosrymer) [inline kelp.js:61]: Nice work switching on the array!

**Comment by yaraaguilar** [comment]
I added a new file urchin.js

**Pull request closed without merging**